stdlib/3/collections/__init__.pyi

```diff
@@ -297,11 +297,19 @@ class defaultdict(Dict[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, default_factory: Optional[Callable[[], _VT]]) -> None: ...
     @overload
+    def __init__(self, default_factory: Optional[Callable[[], _VT]], **kwargs: _VT) -> None: ...
+    @overload
     def __init__(self, default_factory: Optional[Callable[[], _VT]],
                  map: Mapping[_KT, _VT]) -> None: ...
     @overload
+    def __init__(self, default_factory: Optional[Callable[[], _VT]],
+                 map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
+    @overload
     def __init__(self, default_factory: Optional[Callable[[], _VT]],
                  iterable: Iterable[Tuple[_KT, _VT]]) -> None: ...
+    @overload
+    def __init__(self, default_factory: Optional[Callable[[], _VT]],
```

Comment:
I don't think this one works:
```
In [29]: defaultdict(int, {'b': 4}, [('c', 5)], a=3)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~/tmp.py in <module>()
----> 1 defaultdict(int, {'b': 4}, [('c', 5)], a=3)

TypeError: dict expected at most 1 arguments, got 2
```

Reply:
This overload does not take a `Mapping`. There is no overload defined that takes both a `Mapping` and an `Iterable`. mypy does report an error on that line.